core/templates/pages/exploration-editor-page/history-tab/history-tab.component.html

```diff
@@ -36,13 +36,16 @@ <h3 class="history-card-header-text">History</h3>
         <i class="fa fa-search"></i>
       </div>
       <div class="history-table-header">
+        <div class="history-table-header-version">
+          <span>Ver. #</span>
```

Comment:
Let's call it "Version No." I think there's enough space (the "Notes" column can be made a bit smaller).

Reply:
sg. done.
![image](https://user-images.githubusercontent.com/11008603/101673176-d4793100-3a7c-11eb-93d2-8ad08707439a.png)